test/snippet/io/alignment_file/begin_iterator.cpp

```diff
@@ -1,16 +1,18 @@
-//! [all]
-#include <seqan3/io/alignment_file/all.hpp>
+#include <string>
+#include <tuple>
 
-using namespace seqan3;
+#include <seqan3/alphabet/nucleotide/dna5.hpp>
+#include <seqan3/io/alignment_file/output.hpp>
+#include <seqan3/std/filesystem>
 
 int main()
 {
-    alignment_file_output fout{std::filesystem::temp_directory_path()/"my.sam"};
+    seqan3::alignment_file_output fout{std::filesystem::temp_directory_path()/"my.sam"};
```

Comment:
stream

Reply:
Should this one be a stream? The snippet references "Returns an iterator to current position in the file."

If it's a file iterator, does it make sense to use begin() on a stream?